sklearn/metrics/_dist_metrics.pyx.tp

```diff
@@ -1219,8 +1248,8 @@ cdef class DatasetsPair:
 
         if not(X.dtype == Y.dtype == np.float64):
             raise ValueError(
-                f"Only 64bit float datasets are supported at this time, "
-                f"got: X.dtype={X.dtype} and Y.dtype={Y.dtype}"
+                f"Datasets must be of np.float64 type. "
+                f"Currently: X.dtype={X.dtype} and Y.dtype={Y.dtype}."
```

Comment:
I kind of preferred the previous version. What's the reason for this change ?

Reply:
I can't remember exactly why. Let's revert.